### PR TITLE
PF-468 Add GoogleAiNotebookUid to the cloud resources uids.

### PR DESCRIPTION
**Pull request by wchamber**
Adding this first to update the Janitor and its client library to use in clean up when creating AI notebook instances.

### CODE REVIEW

File: cloud-resource-schema/src/main/resources/cloud_resources_uid.yaml [hunk 1]
```diff
@@ -10,6 +10,19 @@ paths: {}
...
+      required:
+        - projectId
+        - location
+        - instanceId
+      properties:
+        projectId:
+          type: string
+        location:
+          type: string
+        instanceId:
```
Comment (jaycarlton) [inline cloud-resource-schema/src/main/resources/cloud_resources_uid.yaml:24]: maybe add descriptions to these. I'm not familiar with the instance ID format.

**Review by jaycarlton — APPROVED** [review]

File: cloud-resource-schema/src/main/resources/cloud_resources_uid.yaml (hunk 1 above)
Comment (wchamber) [reply cloud-resource-schema/src/main/resources/cloud_resources_uid.yaml:25]: took a stab, ptal. Like most things in CRL, needs some knowledge of the underlying objects https://cloud.google.com/ai-platform/notebooks/docs/reference/rest/v1/projects.locations.instances

**Review by yonghaoy — APPROVED** [review]

**Review by yonghaoy — COMMENTED** [review]
And also bump schema version

File: cloud-resource-schema/src/main/resources/cloud_resources_uid.yaml (hunk 1 above)
Comment (jaycarlton) [reply cloud-resource-schema/src/main/resources/cloud_resources_uid.yaml:25]: thanks

**Comment by wchamber** [comment]
> And also bump schema version

sync'd offline, that's the `cloud-resource-schema/gradle.properties` which we are updating.

**Pull request merged**